flang/include/flang/Tools/CLOptions.inc

```diff
@@ -70,40 +70,16 @@ inline void addCSE(mlir::PassManager &pm) {
       pm, disableFirCse, fir::createCSEPass);
 }
 
-#if !defined(FLANG_EXCLUDE_CODEGEN)
```

Comment:
Why are these moved? Seems like gratuitous change for the sake of change.

Reply:
Only `createBasicFirPassPipeline` is moved. That's required in order for `bbc` to be able to use it. Methods for code-gen are [excluded](https://github.com/flang-compiler/f18-llvm-project/blob/fir-dev/flang/tools/bbc/bbc.cpp#L133-L134) in `bbc`. However, with this change `createBasicFirPassPipeline` becomes a method that we do want to use in `bbc` (I removed code-gen passes from there).

The diff suggests that other methods are moved as well, but that's how `git diff` is displayed. There isn't much I can do about this.